src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -332,9 +332,14 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             return $scope.submission.actionLogs[index].entry;
         };
 
-        // TODO: determine how this is set
         var isUmiRelease = function() {
-            return 'no';
+            var umiRelease = 'no';
+            var umiReleaseFilterFieldValue = $scope.submission.fieldValues.filter(function (fv) {
+                if(fv.fieldPredicate.value === "umi_publication") {
+                    umiRelease = fv.value === 'true' ? 'yes' : 'no';
```

Comment:
once again sorry, can you break out of the for loop when field value with predicate umi_publication is found

break;